src/pages/StatisticsPage.tsx

```diff
@@ -0,0 +1,160 @@
+import { useEffect, useState } from 'react';
+import BottomNavBar from '@components/common/BottomNavBar.tsx';
+import StatsCard from '@components/statistics/StatsCard.tsx';
+import StatsGraph from '@components/statistics/StatsGrpah.tsx';
+import useMonthlyAnalysis from '@hooks/useMonthlyAnalysis.tsx';
+import styled from '@emotion/styled';
+import { GGAMJA_COLOR } from '../styles/Colors.ts';
+import type { CardCategory } from '@@types/index.ts';
+import { CARD_CATEGORY_KO } from '@utils/index.ts';
+
+const PageWrapper = styled.div`
+  background-color: #faf8f5;
+  color: ${GGAMJA_COLOR.DARK_BROWN};
+  padding-bottom: 80px;
+  height: 100%;
+  overflow: auto;
+`;
+
+const Container = styled.div`
+  padding: 40px 20px 20px 20px;
+`;
+
+const Header = styled.header`
+  text-align: center;
+`;
+
+const PageTitle = styled.h1`
+  font-size: 36px;
+  font-weight: 700;
+  color: ${GGAMJA_COLOR.DARK_BROWN};
+`;
+
+const StatisticsContentWrapper = styled.div`
+  border: 1px solid ${GGAMJA_COLOR.DARK_BROWN};
+  border-radius: 10px;
+  padding: 40px 20px 40px 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  aspect-ratio: 1 / 1;
+  box-shadow: 4px 4px 0 ${GGAMJA_COLOR.DARK_BROWN};
+  gap: 10px;
+`;
+
+const StatsCardGridContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  gap: 20px;
+  width: 100%;
+  align-items: center;
+`;
+
+const StatisticsContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid ${GGAMJA_COLOR.DARK_BROWN};
+  border-radius: 10px;
+  gap: 10px;
+  padding: 20px;
+  margin-bottom: 20px;
+  box-shadow: 4px 4px 0 ${GGAMJA_COLOR.DARK_BROWN};
+`;
+
+const StatisticsTitle = styled.h2`
+  font-size: 24px;
+  font-weight: 700;
+  width: 100%;
+  color: ${GGAMJA_COLOR.DARK_BROWN};
+  text-align: left;
+`;
+
+const StatisticsTextContainer = styled.div<{ isSurpassed: boolean }>`
+  font-size: 20px;
+  font-weight: 400;
+  width: 100%;
+  color: ${GGAMJA_COLOR.DARK_BROWN};
+`;
+
+const StatisticsCategory = styled.h3`
+  display: inline;
+  font-weight: 700;
+`;
+
+const StatisticsPage = () => {
+  const { fetchMonthlyAnalysis } = useMonthlyAnalysis();
+  const [nickname, setNickname] = useState<string>('사용자');
+  const [categories, setCategories] = useState<{ category: CardCategory; quizLogCount: number; cardLogCount: number; accuracy: number }[]>([]);
+  const [strengths, setStrengths] = useState<string[]>([]);
+  const [weaknesses, setWeaknesses] = useState<string[]>([]);
```

Comment:
```suggestion
  const [strengths, setStrengths] = useState<CardCategory[]>([]);
  const [weaknesses, setWeaknesses] = useState<CardCategory[]>([]);
```

이렇게 바꾸면 아래에서 category as CardCategory 부분에서  as CardCategory를 제거할 수 있습니다.